awx/ui_next/src/App.jsx

```diff
@@ -29,7 +37,7 @@ const ProtectedRoute = ({ children, ...rest }) =>
   );
 
 function App() {
-  const catalogs = { en, ja };
+  const catalogs = { en: english, ja: japanese, zu: zulu, fr: french };
```

Comment:
Do we need to add the other just added languages also? 🇪🇸 🇨🇳 🇳🇱

Reply:
based on the convo we had today at dev sync I am trying to figure out the best approach here. In production, yes I think we will need to add all the language catalogs, except `zu`.